### PR TITLE
Optimizing size of Extended Access i18n strings

### PR DESCRIPTION
**Pull request by ChrisAntaki**
- Relocates `build-i18n` script, so it's part of the `build-system` instead of the `assets`
- Removes unnecessary `assets/i18n/strings` folder
- Updates Extended Access i18n strings
- Updated Extended Access regwall to support more efficient i18n strings
- Simplifies a Swgjs build step

## Bundle size diffs (gzipped)
| Bundle | Before | After | Diff |
| :--- | :---: | :---: | :---: |
| Swgjs Extended Access | 17,811 B |  17,481 B | -330 B |

### CODE REVIEW

File: assets/i18n/strings/compile.js [hunk 1]
```diff
@@ -36,7 +36,14 @@ async function main() {
       if (!localesPerMessage[name]) {
         localesPerMessage[name] = {};
       }
-      localesPerMessage[name][locale] = $msg.html();
+      localesPerMessage[name][locale] = $msg
+        .html()
+        // Simplify template.
+        .replace('<strong></strong>', '')
```
Comment (hartmannr76) [inline assets/i18n/strings/compile.js:42]: Aren't the strings exported from our own tools? Can't we update our exporter to not include the tag if it isn't necessary? I'd prefer that over updating our compiler for things like this

Comment (ChrisAntaki) [reply assets/i18n/strings/compile.js]: Will do!

File: build-system/tasks/build-i18n.js [hunk 2]
```diff
@@ -16,15 +16,19 @@
...
+// https://docs.google.com/document/d/1FMEKJ_TmjHhqON0krE4xhDbTEj0I0DnvzxMzB2cWUWA/edit?resourcekey=0-TQ7hPOzAD4hX8x9PfweGSg#heading=h.q9gi7t4h1tyj
+
 const cheerio = require('cheerio');
 const fs = require('fs').promises;
 
-async function main() {
-  const files = await fs.readdir(__dirname);
+async function buildI18nStrings() {
+  const xlbDir = __dirname + '/../../assets/i18n/strings';
+  const files = await fs.readdir(xlbDir);
```
[the remark this answers is not in the document]
Comment (ChrisAntaki) [reply build-system/tasks/build-i18n.js:27]: There's a g3 script that now saves the files there, runs the `build-i18n` npm script, then deletes the "assets/i18n/strings" folder

File: build-system/tasks/build-i18n.js [hunk 3]
```diff
@@ -16,15 +16,19 @@
 
 /** @fileoverview Compiles XLB files into a single JSON file with translations. */
 
+// Note: Want to change Extended Access i18n strings? Check out this document:
+// https://docs.google.com/document/d/1FMEKJ_TmjHhqON0krE4xhDbTEj0I0DnvzxMzB2cWUWA/edit?resourcekey=0-TQ7hPOzAD4hX8x9PfweGSg#heading=h.q9gi7t4h1tyj
```
[the remark this answers is not in the document]
Comment (ChrisAntaki) [reply build-system/tasks/build-i18n.js]: Updated!

**Review by fikus — APPROVED** [review]

**Review by hartmannr76 — APPROVED** [review]

**Pull request merged**